### PR TITLE
Auto-update charm libraries

### PR DESCRIPTION
**Pull request by arturo-seijas**

### CODE REVIEW

File: .github/workflows/auto_update_libs.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Auto-update charm libraries
```
Comment (amandahla) [inline .github/workflows/auto_update_libs.yaml:1]: I've just realized something: do you think it would be useful to add a badge for this workflow?

Comment (arturo-seijas) [reply .github/workflows/auto_update_libs.yaml:1]: Not sure if it makes sense. What do you think?

**Review by nrobinaubertin — APPROVED** [review]

File: .github/workflows/auto_update_libs.yaml (hunk 1 above)
Comment (arturo-seijas) [reply .github/workflows/auto_update_libs.yaml:1]: Anyway, I'd say that's out of the scope of this PR, but I'm happy to discuss it further :)

**Review by gtrkiller — APPROVED** [review]
LGTM

**Pull request merged**